ccdproc/astrometry.py

```diff
@@ -0,0 +1,478 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# This module implements the combiner class.
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+
+import itertools as it
+import numpy as np
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+from astropy import wcs
+from astropy.modeling import models
+from astropy.modeling import fitting
+
+__all__ = ['remove_duplicates', 'distance', 'distance_ratios',
+           'triangle_angle', 'calc_triangle', 'match_by_triangle',
+           'match_by_fit', 'create_wcs_from_fit']
+
+
+def remove_duplicates(x, y, tolerance):
+    """Remove duplicates within a certain tolerance from a pair of coordiantes.
+
+    Parameters
+    ----------
+    x : `numpy.ndarray`
+        x-position of objects
+
+    y : `numpy.ndarray`
+        y-position of objects
+
+    tolerance : float
+        tolerance for removal of dupliactes
+
+    Returns
+    -------
+    x : `numpy.ndarray`
+        x-position of objects with duplicates removed
+
+    y : `numpy.ndarray`
+        y-position of objects with duplicates removed
+    """
+    mask = np.ones(len(x), dtype=bool)
+
+    for i in range(len(x)):
+        if mask[i]:
+            d = ((x - x[i])**2 + (y - y[i])**2)**0.5
```

Comment:
You could use the `distance` function here.

Reply:
will replace with np.hypot